### PR TITLE
docs: use a custom directory for serving static files

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- docs: use a custom directory for serving static files (https://github.com/VeryGoodOpenSource/dart_frog/issues/309)

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/docs/basics/serving-static-files.md [hunk 1]
```diff
@@ -33,3 +37,25 @@ In production, only files that are in the `/public` directory at build time will
 :::caution
 Be sure not to have a static file with the same name as a file in the `/routes` directory as this will result in a conflict.
 :::
+
+## Using a Custom Directory ✨
+
+Even though Dart Frog uses the `public` directory for serving static files by default, you can also specify a custom directory by creating a [custom entrypoint](/docs/advanced/custom_entrypoint).
+
+Create a `main.dart` at the root of your project with the following contents:
```
Comment (erickzanardo) [inline docs/docs/basics/serving-static-files.md:45]: Shouldn't this be in the singular?

```suggestion
Create a `main.dart` at the root of your project with the following content:
```

**Review by erickzanardo — APPROVED** [review]
One nit/question, but LGTM already

File: docs/docs/basics/serving-static-files.md (hunk 1 above)
Comment (felangel) [reply docs/docs/basics/serving-static-files.md:45]: I think technically either is fine but looks like file contents is more commonly use to refer to this
https://english.stackexchange.com/questions/56831/file-content-vs-file-contents

Comment (erickzanardo) [reply docs/docs/basics/serving-static-files.md:45]: Interesting! Thanks for explanation

**Pull request merged**